schutzbot/Jenkinsfile

```diff
@@ -291,6 +302,17 @@ pipeline {
                         }
                     }
                 }
+                stage('EL8: koji-osbuild') {
+                    agent { label "rhel8cloudbase && psi && x86_64" }
```

Comment:
The same as in the previous comment.

Reply:
The RHEL tests do need access, and I think `psi` is already present here, no?

Reply:
Yep, leaving as is with `psi` in there then.